build/helpers/moment.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var moment = require('moment');
+
+module.exports = function (context, options) {
+	var date = options ? context : new Date();
+	var options = options || context;
+	var format = options.hash.format || 'YYYY-MM-DD ';
+	// http://momentjs.com/docs/#/displaying/
+	return moment(date).utc().format(format);
```

Comment:
Some background from the example I provided...

I don't claim to understand _why_, but unless you convert to utc time, my experience is that this will _always_ be wrong (always much earlier, so sometimes the output will be a day earlier than intended... `2014-04-26` instead of `2014-04-27`, for example).

It felt hack-ish to always convert the time to `utc` (even though that's what a few StackOverflow things told me to do), but then I saw [metalsmith-permalinks](https://github.com/segmentio/metalsmith-permalinks/blob/master/lib/index.js#L93) does the same thing, so I decided I'd stop questioning what worked and move on.